processdoc.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+# Usage: $0 <foo.md> <out-foo.frontmatter> <out-foo-processed.md>
+
+import sys
+import re
+from visidata import vd, AttrDict, Sheet
+
+# copied from visidata/guide.py
+class OptionHelpGetter:
+    'For easy and consistent formatting in sidebars and helpstrings, use {help.options.opt_name}.'
+    def __getattr__(self, optname):
+        opt = vd.options._get(optname, 'default')
+        return f'<a href="/man#options-{optname}">`{optname}`</a>: {opt.helpstr} (default: {opt.value})'
+
+
+class CommandHelpGetter:
+    'For easy and consistent formatting in sidebars and helpstrings, use {help.commands.long_name}.'
+    def __init__(self, cls):
+        self.cls = cls
+        self.helpsheet = vd.HelpSheet()
+        self.helpsheet.ensureLoaded()
+
+    def __getattr__(self, k):
+        return self.__getitem__(k)
+
+    def __getitem__(self, k):
+        longname = k.replace('_', '-')
+        binding = self.helpsheet.revbinds.get(longname, [None])[0]
```

Comment:
I ran into an issue while writing a JSON guide and tried to [address it](https://github.com/saulpw/visidata/pull/2268/files#diff-2c3a790d477d55deb5ed1617c7c2802245ae9eb8c35f04bd7104063c0d8f2658L125-R125) in saulpw/visidata#2268 by defining a fallback string when commands have no keybindings. Not sure what the right/best way is to handle it, but I'm guessing any update would need to happen in both vd's guide.py and here?

```suggestion
        binding = self.helpsheet.revbinds.get(longname, [None])[0] or '<unbound>'
```
```

Reply:
Yes, good point!  We'll get your PR reviewed here shortly and make sure they're in sync.

Reply:
Cool cool, no rush on that. And I can certainly split the unbound-command fix into a separate smaller/more-focused pr if that's useful.

Reply:
Oh, I would love it if you seperated it out! I don't want its merging to be blocked on reviewing the json text. =) That seems really useful.

Reply:
Sure thing - saulpw/visidata#2273